src/components/ValuesScreen.tsx

```diff
@@ -1,28 +1,127 @@
 import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
+import { View, StyleSheet } from 'react-native';
+import { Text, Title, List, Button} from 'react-native-paper';
 
 import { createStackNavigator } from '@react-navigation/stack';
 import { CustomNavigationBar } from './CustomNavigationBar';
+import { FlatList } from 'react-native-gesture-handler';
 
 const ValuesStack = createStackNavigator();
 
-const ViewContent = () => {
+const RelationView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Relation</Text>
+  </View>
+); 
+
+const CareerView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Studier/karrriär</Text>
+  </View>
+); 
+
+const InterestsView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Fritid/intressen</Text>
+  </View>
+);
+
+const MindView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Sinne/Kropp/Spirituellt</Text>
+  </View>
+);
+
+const ResposibilityView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Dagliga ansvar</Text>
+  </View>
+);
+
+const SupportView = () => (
+  <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+    <Text>Stödpersoner</Text>
+  </View>
+);
+
+
+const CreateButton = (props: any) => {
```

Comment:
```suggestion
const StyledButton = (props: any) => {
```
"CreateButton" sounds more like a function name than a component name. A name similar to my suggestion would be more fitting.